cmd/fleet/main.go

```diff
@@ -538,7 +538,7 @@ LOOP:
 
 		select {
 		case newCfg = <-f.cfgCh:
-			log.Info().Msg("Server configuration update")
+			log.Info().Interface("cfg", redactConfig(curCfg)).Msg("Server configuration update")
```

Comment:
alternatively we can use the `redactConfig` call instead of `redactServerCfg` on lines 643 & 659 in `configChangedServer` (where it logs old/new config)